login-workflow/docs/services.md

```diff
@@ -50,6 +50,12 @@ constructor(pxbAuthConfig: PxbAuthConfig) {
 -   **customEmailValidator** (optional): _`ValidatorFn`_
     -   Custom email regex requirements.
     -   Default: `undefined`
+-   **customFirstNameRequirements** (optional): _`{ maxLength: number }`_
+    -   Custom first name requirements.
+    -   Default: `{ maxLength: undefined }`
+-   **customLastNameRequirements** (optional): _`{ maxLength: number }`_
+    -   Custom last name requirements.
+    -   Default: `{ maxLength: undefined }`
```

Comment:
I'm still imagining maxLength would be an optional prop in this type and the default would be `{ }` (nothing specified)

Reply:
Ok this is updated now; default is now `undefined`